forms/FormFieldSchemaTrait.php

```diff
@@ -85,6 +115,14 @@ public function getSchemaData() {
 		return array_merge($this->getSchemaDataDefaults(), $this->schemaData);
 	}
 
+	public function getSchemaDataType() {
```

Comment:
I can't see a setter for this and I don't understand how it's supposed to be set. Can we add a setter or explain in the docblock above how this gets set?